lib/floe/workflow/states/pass.rb

```diff
@@ -13,6 +13,9 @@ def initialize(workflow, name, payload)
           @end         = !!payload["End"]
           @result      = payload["Result"]
 
+          raise Floe::InvalidWorkflowError, "Missing \"Next\" field in state [#{name}]" if @next.nil? && !@end
```

Comment:
We might want a check whether End is actually defined since above we only do `!!`, meaning the key may not exist?   Not sure if that actually matters or not.

Reply:
That was intentional actually,  `End` missing implicitly means the state is not a terminal state which means `Next` is required.